ocaml/xapi/system_domains.ml

```diff
@@ -78,6 +78,8 @@ let pbd_of_vm ~__context ~vm =
   then Some(Ref.of_string (List.assoc storage_driver_domain_key other_config))
   else None
 
+let driver_domain_ssid = ref 0x44525644l
```

Comment:
is there any justification as to why this wouldn't clash with the SSID of some ordinary domain? or is this a magic number from xen?

Reply:
ordinary domains get a ssid of 0 always. It is a magic number until we have proper Flask security policy support in Xen. (at which point we can refer to the policy instead of hardcoding it in the source code).